tests/python_tests/test_eltwise_binary.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+import torch
+import os
+from helpers import *
+
+def generate_golden(operation, operand1, operand2, data_format):
+    if( data_format == "Float16" or data_format == "Float16_b"):
+        tensor1_float = operand1.clone().detach().to(format_dict[data_format])
+        tensor2_float = operand2.clone().detach().to(format_dict[data_format])
+    else:
+        tensor1_float = operand1.clone().detach().to(format_dict["Float16_b"])
+        tensor2_float = operand2.clone().detach().to(format_dict["Float16_b"])
+    
+    operations = {
+        "elwadd": tensor1_float + tensor2_float,
+        "elwsub": tensor1_float - tensor2_float,
+        "elwmul": tensor1_float * tensor2_float,
+        "elwdiv": tensor1_float / tensor2_float
+    }
+    
+    if operation not in operations:
+        raise ValueError("Unsupported operation!")
+
+    return operations[operation].tolist()
+
+@pytest.mark.parametrize("format", ["Bfp8_b", "Float16_b", "Float16"])
+@pytest.mark.parametrize("testname", ["eltwise_binary_test"])
+@pytest.mark.parametrize("mathop", ["elwadd", "elwsub", "elwmul"])
+@pytest.mark.parametrize("dest_acc", ["DEST_ACC", ""])
+def test_all(format, mathop, testname, dest_acc):
+    
+    src_A, src_B = generate_stimuli(format)
+    golden = generate_golden(mathop, src_A, src_B, format)
+    write_stimuli_to_l1(src_A, src_B, format)
+
+    test_config = {
+        "input_format": format,
+        "output_format": format,
+        "testname": testname,
+        "dest_acc": dest_acc,
+        "mathop": mathop
+    }
+
+    make_cmd = generate_make_command(test_config)
+    run_shell_command(f"cd .. && {make_cmd} >/dev/null")
```

Comment:
```suggestion
    run_shell_command(f"cd .. && {make_cmd}")
```
No need for `> /dev/null` since you have `capture_output` set to `False`. Applies for other invocations where you set redirection to /dev/null.

Reply:
It reduces amount of stuff printed in terminal. For example messages like this:

build/tmu-crt0.o
build/main_unpack.o
build/eltwise_unary_datacopy_test_unpack.o
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libstdc++.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libm.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/libgcc.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libc.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libgloss.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libc.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libgloss.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/libgcc.a
build/tmu-crt0.o
build/main_math.o
build/eltwise_unary_datacopy_test_math.o
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libstdc++.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libm.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/libgcc.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libc.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libgloss.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libc.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libgloss.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/libgcc.a
build/tmu-crt0.o
build/main_pack.o
build/eltwise_unary_datacopy_test_pack.o
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libstdc++.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libm.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/libgcc.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libc.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libgloss.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libc.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libgloss.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/libgcc.a
build/tmu-crt0.o
build/brisc.o
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libstdc++.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libm.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/libgcc.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libc.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libgloss.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libc.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/../../../../riscv32-unknown-elf/lib/libgloss.a
/localdev/ldjurovic/tt-llk/tests/sfpi/compiler/bin/../lib/gcc/riscv32-unknown-elf/12.2.0/libgcc.a

won't be printed to terminal

Reply:
Ah, you are right. I was missing `stdout=subprocess.DEVNULL` parameter in the subprocess.run. Please add it, and then you won't need to use `/dev/null` explicitly.

Reply:
We do however want to keep the stderr prints.